GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -179,14 +180,14 @@ private bool TryRunUpgrade(out Version newVersion, out string consoleError)
                 },
                 "Downloading"))
             {
-                consoleError = errorMessage;
+                consoleError = error;
                 return false;
             }
 
             if (!this.LaunchInsideSpinner(
                 () =>
                 {
-                    if (!this.preRunChecker.TryUnmountAllGVFSRepos(out errorMessage))
+                    if (!this.preRunChecker.TryUnmountAllGVFSRepos(out error))

```

Comment:
Reading through the code - it feels like this code to mount and unmount should be next to each other.

```
TryUnmount(...);
try
{
    DoWork(...);
}
finally
{
    TryMount(...);
}

Reply:
As discussed, this is un-related to current Nuget refactoring. Deferring to a future PR.